### PR TITLE
PLANET-6091 Align blocks with page width

### PR DESCRIPTION
**Pull request by mleray**
### Description

See [PLANET-6091](https://jira.greenpeace.org/browse/PLANET-6091)

- There used to be a small width difference between the CarouselHeader slides content and the rest of the page content.
- The Columns block also used to be slightly different because its content was put outside of the `container` div.
- The Gallery block was missing the `container` block.

We also need to update the donation pages, there is another [PR](https://github.com/greenpeace/planet4-en-petitions/pull/8) for that.

Main PR: [master-theme](https://github.com/greenpeace/planet4-master-theme/pull/1508)

### Testing

You can test the changes on this [test instance](https://www-dev.greenpeace.org/test-jupiter/) or on local.

### CODE REVIEW

File: assets/src/blocks/CarouselHeader/CarouselControls.js [hunk 1]
```diff
@@ -20,21 +20,17 @@ export const CarouselControls = ({
     {/* Indicators */}
     <div className='carousel-indicators-wrapper'>
       <div className='container'>
-        <div className='row'>
```
Comment (mleray) [inline assets/src/blocks/CarouselHeader/CarouselControls.js:23]: These classes `row` and `col` didn't seem to add anything to the layout, so I removed them

**Comment by planet-4** [comment]
### Test instance is ready :rocket:

:new_moon: [jupiter](https://www-dev.greenpeace.org/test-jupiter) | [admin](https://www-dev.greenpeace.org/test-jupiter/wp-admin/) | [blocks report](https://www-dev.greenpeace.org/test-jupiter/wp-admin/?page=plugin_blocks_report) | [CircleCI](https://app.circleci.com/pipelines/github/greenpeace/planet4-test-jupiter) | [composer-local.json](https://github.com/greenpeace/planet4-test-jupiter/blob/main/composer-local.json)

:watch: 2021.11.10 10:54:10

**Review by lithrel — APPROVED** [review]
:+1: 
About the Gallery carousel style, the xxl breakpoint is the only one where the height doesn't gain 100px (cf https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/blob/86652ee6e15938e6ff1781a73ed2f109f5d935eb/assets/src/styles/blocks/Gallery/styles/GalleryCarousel.scss#L27-L45 ) which changes a bit the view of the image, don't know if it's expected

**Comment by mleray** [comment]
> 👍 About the Gallery carousel style, the xxl breakpoint is the only one where the height doesn't gain 100px

That's a good point, I'll double-check with Houssam!

**Review by lithrel — APPROVED** [review]
:+1:

**Pull request merged**